test/plugin/in_tail/test_position_file.rb

```diff
@@ -282,4 +282,29 @@ def build_files(file)
       assert_equal 2, f.read_inode
     end
   end
+
+  sub_test_case "TargetInfo" do
+    def test_equal
+      t1 = Fluent::Plugin::TailInput::TargetInfo.new("test", 1234)
+      t2 = Fluent::Plugin::TailInput::TargetInfo.new("test", 1235)
+
+      assert_equal t1, t2
+    end
+
+    def test_eql?
+      t1 = Fluent::Plugin::TailInput::TargetInfo.new("test", 1234)
+      t2 = Fluent::Plugin::TailInput::TargetInfo.new("test", 5321)
+
+      assert do
+        t1.eql? t2
+      end
+    end
+
```

Comment:
To emphasize the difference of eql? and equal? behavior, how about adding one more test case using `assert_not_same t1, t2`

Reply:
This should be done in https://github.com/fluent/fluentd/pull/3391/commits/ab36738b86e908cd9a5867a704008bd687b9a1eb and https://github.com/fluent/fluentd/pull/3391/commits/2f2214b5f0840990227ad2fcab709c1e9092f657.